README.md

```diff
@@ -26,61 +28,130 @@ Please feel free to [pull requests](https://github.com/he-y/awesome-Pruning/pull
 
 ### Type of Pruning
 
-| Type        | `F`            | `W`            | `Other`     |
-|:----------- |:--------------:|:--------------:|:-----------:|
-| Explanation | Filter pruning | Weight pruning | other types |
+| Type        | `F`            | `W`            | `S`              | `Other`     |
+|:----------- |:--------------:|:--------------:|:----------------:|:-----------:|
+| Explanation | Filter pruning | Weight pruning | Special Networks | other types |
+
```

Comment:
add "A survey for filter pruning" section.
Main text:
A survey of structured pruning can be found at this link XX. The related papers are categorized as below:
(then upload the "tree" of the survey)

Reply:
link is arxiv link